code/datums/keybinding/xeno.dm

```diff
@@ -224,6 +224,13 @@
 	keybind_signal = COMSIG_XENOABILITY_CREATE_BOMB
 	hotkey_keys = list("F")
 
+/datum/keybinding/xeno/root
+	name = "root"
+	full_name = "Boiler: Root in place"
+	description = ""
```

Comment:
missing desc

Reply:
The majority of the binds are missing descs, I'll do the ones for boiler abilities but that's a todo right there